### PR TITLE
Standardise help command

### PR DESCRIPTION
**Pull request by yixiann**
Modify UI to match main application

image

Add chicken
image

Fixes : https://github.com/AY2223S1-CS2103T-W16-2/tp/issues/240

### CODE REVIEW

File: src/main/java/seedu/foodrem/commons/enums/CommandType.java [hunk 1]
```diff
@@ -16,209 +16,244 @@
  */
 public enum CommandType {
```
Comment (RichDom2185) [inline src/main/java/seedu/foodrem/commons/enums/CommandType.java:17]: Why was this file reorganised again? Was there any reason to do it? I reorganised it last time during refactoring because I thought organising them in alphabetical order would make it easier for developers to quickly jump to a section.

File: copyright.txt [hunk 2]
```diff
@@ -1,9 +1,5 @@
 ﻿Some code adapted from http://code.makery.ch/library/javafx-8-tutorial/ by Marco Jakob
```
Comment (RichDom2185) [inline copyright.txt:2]: Can we add a copyright acknowledgement for the FoodRem chicken icon?

File: src/main/resources/view/HelpWindow.css [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
 
 #copyButton:hover {
-  -fx-background-color: gray;
-}
-
-#copyButton:armed {
-  -fx-background-color: darkgray;
+  -fx-background-color: #0096C9;
+  -fx-text-fill: #ffffff;
 }
```
Comment (RichDom2185) [inline src/main/resources/view/HelpWindow.css:11]: Let's add a TODO for me to standardise the font.

```suggestion

/* TODO: Richard: Standardise font */
```

**Review by RichDom2185 — COMMENTED** [review]
There are some comments I would like to seek clarification for, but otherwise LGTM, thanks for helping with the UI!

File: src/main/java/seedu/foodrem/commons/enums/CommandType.java (hunk 1 above)
Comment (yixiann) [reply src/main/java/seedu/foodrem/commons/enums/CommandType.java:17]: I did not know it was in alphabetical order, I arranged it in terms of type of commands i.e. general commands, item commands and tag commands.

Comment (yixiann) [reply src/main/java/seedu/foodrem/commons/enums/CommandType.java:17]: I arranged it in the order the commands I felt will be used in.

File: copyright.txt (hunk 2 above)
Comment (yixiann) [reply copyright.txt:2]: image

Comment (yixiann) [reply copyright.txt:2]: image

Comment (yixiann) [reply copyright.txt:2]: Im not sure if I understand it correctly but I do not think figma has copyright

File: src/main/java/seedu/foodrem/commons/enums/CommandType.java (hunk 1 above)
Comment (RichDom2185) [reply src/main/java/seedu/foodrem/commons/enums/CommandType.java:17]: As in, I also arranged into the 3 groups
- General commands
- Item commands
- Tag command

But within each group, the commands are in alphabetical order cause I thought it would be easier that way,

File: copyright.txt (hunk 2 above)
Comment (RichDom2185) [reply copyright.txt:2]: Creative Commons **Atribution** 4.0 means you can freely adapt as long as you give credit (attribution) to the original creator. [More Info](https://smartcopying.edu.au/quick-guide-to-creative-commons/).

File: src/main/java/seedu/foodrem/commons/enums/CommandType.java (hunk 1 above)
Comment (yixiann) [reply src/main/java/seedu/foodrem/commons/enums/CommandType.java:18]: Ah I understand

Comment (yixiann) [reply src/main/java/seedu/foodrem/commons/enums/CommandType.java:18]: Will rearrange. Thanks for bringing this up

File: src/main/java/seedu/foodrem/logic/parser/CliSyntax.java [hunk 4]
```diff
@@ -12,4 +12,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ITEM_BOUGHT_DATE = new Prefix("bgt/");
     public static final Prefix PREFIX_ITEM_EXPIRY_DATE = new Prefix("exp/");
     public static final Prefix PREFIX_ITEM_PRICE = new Prefix("p/");
+    public static final Prefix PREFIX_ITEM_REMARKS = new Prefix("r/");
+    public static final Prefix PREFIX_ID = new Prefix("id/");
```
Comment (RichDom2185) [inline src/main/java/seedu/foodrem/logic/parser/CliSyntax.java:16]: Why was this added back?

Comment (RichDom2185) [reply src/main/java/seedu/foodrem/logic/parser/CliSyntax.java:16]: Hmm I see that this is because the recently-merged remarks command requires the `id/`. Let's create an issue to standardise this and remove the `id/` prefix from the remarks command in order to match the other commands. @yixiann Can you open the issue?

Comment (yixiann) [reply src/main/java/seedu/foodrem/logic/parser/CliSyntax.java]: I think it was automatically added back during a merge, will remove. The remarks command do not use id/

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks a lot for the quick fixes!
